src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -41,6 +41,11 @@ public class TextureManager : MonoBehaviour
         /// </summary>
         public RenderTexture compositeTexture { get; private set; }
 
+        /// <summary>
+        /// The texture output to a capture card when quadrant view isn't enabled

```

Comment:
>The texture output to a capture card when quadrant view isn't enabled [](start = 12, length = 69)

The output to the capture card is currently always the same, it doesn't matter if quadrant view is enabled or not